tests/lookup/tests.py

```diff
@@ -552,6 +552,16 @@ def test_in(self):
             ]
         )
 
+    def test_notin(self):
+        self.assertQuerysetEqual(
+            Article.objects.filter(id__notin=[4, 5, 6, 7, 999]),
```

Comment:
I don't think this test demonstrate _why_ this feature is required as this test passes with `exclude(id__in=[4, 5, 6, 7, 999])` and differences between both lookups is the main selling point of the feature.